bokehjs/src/coffee/common/canvas.coffee

```diff
@@ -43,7 +67,7 @@ class CanvasView extends ContinuumView
                           @ctx.mozBackingStorePixelRatio ||
                           @ctx.msBackingStorePixelRatio ||
                           @ctx.oBackingStorePixelRatio ||
-                          @ctx.backingStorePixelRatio || 1
+                          @ctx.backingStorePixelRatio || 1  # ak: wtf?
```

Comment:
Ah, I read this as a list of booleans that were or-ed, and had a 1 in the end, so that the result would always be `true`, but yes... default value ... carry on ... <facepalm>